src/manifolds/Hyperbolic.jl

```diff
@@ -1,116 +1,221 @@
 @doc raw"""
     Hyperbolic{N} <: AbstractEmbeddedManifold{ℝ,DefaultIsometricEmbeddingType}
 
-The hyperbolic space $ℍ^n$ represented by $n+1$-Tuples, i.e. embedded in the
+The hyperbolic space $\mathcal H^n$ represented by $n+1$-Tuples, i.e. embedded in the
 [`Lorentz`](@ref)ian manifold equipped with the [`MinkowskiMetric`](@ref)
 $⟨\cdot,\cdot⟩_{\mathrm{M}}$. The space is defined as
 
 ```math
-ℍ^n = \Bigl\{p ∈ ℝ^{n+1}\ \Big|\ ⟨p,p⟩_{\mathrm{M}}= -p_{n+1}^2
+\mathcal H^n = \Bigl\{p ∈ ℝ^{n+1}\ \Big|\ ⟨p,p⟩_{\mathrm{M}}= -p_{n+1}^2
   + \displaystyle\sum_{k=1}^n p_k^2 = -1, p_{n+1} > 0\Bigr\},.
 ```
 
-The tangent space $T_p ℍ^n$ is given by
+The tangent space $T_p \mathcal H^n$ is given by
 
 ````math
-T_p ℍ^n := \bigl\{
+T_p \mathcal H^n := \bigl\{
 X ∈ ℝ^{n+1} : ⟨p,X⟩_{\mathrm{M}} = 0
 \bigr\}.
 ````
 Note that while the [`MinkowskiMetric`](@ref) renders the [`Lorentz`](@ref) manifold (only)
 pseudo-Riemannian, on the tangent bundle of the Hyperbolic space it induces a Riemannian
 metric. The corresponding sectional curvature is $-1$.
 
+If `p` and `X` are `Vector`s of length `n+1` they are assumed to be
+a [`HyperboloidPoint`](@ref) and a [`HyperboloidTVector`](@ref), respectively
+
+Other models are the Poincaré ball model, see [`PoincareBallPoint`](@ref) and [`PoincareBallTVector`](@ref), respectiely
+and the Poincaré half space model, see [`PoincareHalfSpacePoint`](@ref) and [`PoincareHalfSpaceTVector`](@ref), respectively.
+
 # Constructor
 
     Hyperbolic(n)
 
-Generate the $ℍ^{n} ⊂ ℝ^{n+1}$
+Generate the Hyperbolic manifold of dimension `n`.
 """
 struct Hyperbolic{N} <: AbstractEmbeddedManifold{ℝ,DefaultIsometricEmbeddingType} end
 
 Hyperbolic(n::Int) = Hyperbolic{n}()
 
+@doc raw"""
+    HyperboloidPoint <: MPoint
+
+In the Hyperboloid model of the [`Hyperbolic`](@ref) $\mathcal H^n$ points are represented
+as vectors in $ℝ^{n+1}$ with [`MinkowskiMetric`](@ref) equal to $-1$.
+
+This representation is the default, i.e. vectors are assumed to have this repesentation.
```

Comment:
```suggestion
This representation is the default, i.e. `AbstractVector`s are assumed to have this repesentation.
```
This should be more clear here.